vmm/src/memory_manager.rs

```diff
@@ -1081,8 +1120,13 @@ impl MemoryManager {
             arch_mem_regions,
             ram_allocator,
             dynamic,
+            #[cfg(target_arch = "aarch64")]
+            uefi_flash: None,
         };
 
+        #[cfg(target_arch = "aarch64")]
```

Comment:
How about moving this after the next line `memory_manager.allocate_address_space()?;` ?

Let the common logics go first, then comes the arch specific logic.

Reply:
Ok